pkgs/development/libraries/fontconfig/default.nix

```
@@ -73,7 +74,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     expat
-  ];
+  ] ++ stdenv.lib.optional stdenv.isDarwin CoreFoundation;
```

We don't use `stdenv.lib` anymore. Please use the following and at add `lib` at the top of the file

```suggestion
  ] ++ lib.optional stdenv.isDarwin CoreFoundation;
```